src/views/components/comparison-table.html

```diff
@@ -1,6 +1,13 @@
 <div class="c-comparison-table">
-	<div class="jsdelivr-wrapper"><div></div></div>
+	<div class="jsdelivr-wrapper">
```

Comment:
The tooltips in the tables no longer work.

Reply:
Will check

Reply:
Fixed